charter.md

```diff
@@ -42,3 +44,6 @@ The research group may explore ideas that require such changes, and is uniquely
 placed to discuss their implications with the IETF community, but the potential
 incorporation of such ideas into the standards process is a matter for the IETF
 and is out of scope for this group.
+
+Considerations related to hosts connecting to an NTN and path selection strategies
+among available connectivity (including via NTN) are out of scope.
```

Comment:
Can you say a bit more about what the motivation for this is?  If you're saying "keep all the SD-WAN chatter somewhere else", then I think I agree.  But want to double-check.

Reply:
You got it.